### PR TITLE
adding sample values.yaml for reference

### PR DESCRIPTION
**Pull request by Shubhamag12**
A sample values.yaml for reference

### CODE REVIEW

File: emp/emp-yamls/values.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# A sample values.yaml file for users who want to install EMP related components on EKS cluster manually, using helm chart.
+images: # images used by EMP components
+  eks_vol_watcher:
```
Comment (shamsher31) [inline emp/emp-yamls/values.yaml:3]: Add dummy path to images here

File: emp/emp-yamls/values.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  emp_pod_webhook:
+  evm_autoscaler:
+  evm_vpc_cni:
+  evm_vpc_cni_init:
+manualInstallation: 'true' # (*)
```
Comment (shamsher31) [inline emp/emp-yamls/values.yaml:8]: Do we need single quotes for the boolean value?

Comment (Shubhamag12) [reply emp/emp-yamls/values.yaml]: ah no, my bad

File: emp/emp-yamls/values.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+images: # images used by EMP components
+  eks_vol_watcher: quay.io/platform9/eks-vol-watcher:<version>
+  emp_pod_webhook: quay.io/platform9/emp-pod-webhook:<version>
+  evm_autoscaler: quay.io/platform9/evm-autoscaler:<version>
+  evm_vpc_cni: quay.io/platform9/evm-vpc-cni:<version>
+  evm_vpc_cni_init: quay.io/platform9/evm-vpc-cni-init:<version>
+manualInstallation: true # (*)
+eksVolWatcher: # (*)
+  secretName: <secret-name>
+evmAutoScaler: # (*)
```
Comment (shamsher31) [inline emp/emp-yamls/values.yaml:11]: I think we should remove `# (*)` from all main keys?

File: emp/emp-yamls/values.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  kubeconfig: <redacted>
+  secretName: <secret-name>
+  webhook:
+    namespace: <namespace>
+eksCNI: # (*)
+  secretName: <secret-name>
+applyEMPWebhook: true # this flag determines the installation of EMP Pod Webhook (default: true)
+imagePullSecrets: '' # user needs to populate this field in case images are pushed to a custom registry (note the namespace in which secret is created should be same as specified above)
+
+# (*) denotes that fields are auto-populated by EMP and should not be modified.
```
Comment (shamsher31) [inline emp/emp-yamls/values.yaml:24]: I think we should move this at the top?

**Review by shamsher31 — APPROVED** [review]
LGTM

**Pull request merged**